app.js

```diff
@@ -1,12 +1,16 @@
 const express = require('express');
-const {getCategories, getReviews} = require('./controllers/controller.games');
-const {handle404Errors} = require('./controllers/controller.errors')
+const {getCategories, getReviews, getReviewID} = require('./controllers/controller.games');
+const {handle404Errors, handleOtherErrors, handle500Errors} = require('./error_handling')
```

Comment:
Really like the way your error handling has been split out to a seperate file to keep the app.js readable! 👍